Gododir/gen.go

```diff
@@ -404,14 +414,16 @@ func generate(service *service) {
 	fmt.Printf("Generating %s...\n\n", service.Fullname)
 	delete(service)
 
-	autorest := exec.Command(fmt.Sprintf("%s/autorest/src/core/AutoRest/bin/Debug/net451/win7-x64/autorest", autorestDir),
+	autorest := exec.Command("gulp",
+		"autorest",
 		"-Input", fmt.Sprintf("%s/azure-rest-api-specs/%s.json", swaggersDir, service.Input),
 		"-CodeGenerator", "Go",
 		"-Header", "MICROSOFT_APACHE",
 		"-Namespace", service.Name,
 		"-OutputDirectory", service.Output,
 		"-Modeler", "Swagger",
 		"-pv", sdkVersion)
+	autorest.Dir = fmt.Sprintf("%s/autorest", autorestDir)
```

Comment:
We're all cross-platform like now! How about we update it to:

``` Go
autorest.Dir = filepath.Join(autores.Dir, "autorest")
```

[filepath.Join(string...)](https://golang.org/pkg/path/filepath/#Join)